RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+Changes:
+    * Fixed Run Scanner's interactive Swagger docs generating incorrect URLs in some configurations
```

Comment:
Just noticed there are too many spaces here. You can view the rich diff of your markdown files to make sure they're formatted correctly